daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/svalue/Hasher.scala

```diff
@@ -1,106 +0,0 @@
-// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package com.daml.lf.speedy.svalue
-
-import com.daml.lf.speedy.SValue
-import com.daml.lf.speedy.SValue._
-
-import scala.annotation.tailrec
-import scala.collection.JavaConverters._
-import scala.util.hashing.MurmurHash3
-
-private[speedy] object Hasher {
```

Comment:
@hurryabit, I forget but since we do not use `HashMap` for `STextMap` we can also drop the code to generate the hashcode.

Reply:
:fire: